language/move-prover/src/cli.rs

```diff
@@ -50,6 +51,8 @@ pub struct Options {
     pub verbosity_level: LevelFilter,
     /// Whether to run the documentation generator instead of the prover.
     pub run_docgen: bool,
+    /// Whether to run the ABI generator instead of the prover.
+    pub run_abigen: bool,
```

Comment:
Hmm this is making me think more and more that this should be a part of the `stdlib` crate -- we'll probably want to specify there whether or not a new set of ABIs should be generated as opposed to in the prover.

Reply:
You guys convinced me to do it like this in the thread :)
More seriously, abigen and docgen are quite similar (both called by the stdlib tool, both generate files) so it seems to me that there is broader discussion to have separately.

Reply:
ahhh sorry about that! I absolutely agree that this is a broader discussion in terms of crate placement, and the placement of this is absolutely _not_ a blocker for this PR.